test/cpp/end2end/xds/xds_end2end_test_lib.h

```diff
@@ -232,10 +232,12 @@ class XdsEnd2endTest : public ::testing::TestWithParam<XdsTestType> {
 
     // If use_xds_enabled_server is true, the server will use xDS.
     explicit ServerThread(XdsEnd2endTest* test_obj,
-                          bool use_xds_enabled_server = false)
+                          bool use_xds_enabled_server = false,
+                          bool allow_put_requests = false)
```

Comment:
Seems like we no longer need this parameter here, in `CreateBackends()`, in `CreateAndStartBackends()`, or in the `BackendServerThread` ctor.